src/util.ts

```diff
@@ -112,14 +102,22 @@ export const popoverRectForPosition = (
       }
       break;
     default:
+      top = childRect.top - height - padding;
```

Comment:
Having the default case occur would have meant that top and left are both equal to undefined which are not valid return values for the type of `ClientRect`